source/content/guides/autopilot/05-troubleshoot.md

```diff
@@ -109,10 +109,136 @@ Alternatively, you can add the plugin or theme to the **Excluded Updates** list
+  
+If a merge conflict is preventing you from merging a Multidev environment, follow the steps in the documentation for [Resolving Conflicts from Multidevs](/git-resolve-merge-conflicts#resolve-content-conflicts) and learn how to [Compare Multidev Environments Locally](/multidev#compare-multidev-environments-locally).
+
+If the error is diplayed due to a failed Composer build, use `git diff` to view changes, and examine the error in the log. Composer build logs are only available after the action completes or fails. For more information, refer to the documentation on [Troubleshooting Code Syncs and Upstream Updates
+](/guides/integrated-composer#troubleshooting-code-syncs-and-upstream-updates) and [Adding Dependencies to Your Upstream](/guides/integrated-composer#how-to-add-dependencies-to-your-upstream). 
+
+</Accordion>
+
+## Redirects
+<Accordion title="" id="too-many-redirects" icon="info-sign">
+
+### Issue
+
+The visual regression test (VRT) could not be completed due to a redirect error on one or more pages.
+  
+### Diagnosis 
+
+Autopilot fails when a VRT page redirects more than 8 times or is stuck in an infinite loop.
+
+### Solution
+
+Real-time email notifications are sent for failed VRTs for a site. If an error is detected during an update attempt, the Autopilot Status displays **Needs Attention**  with two options to **Review Test Results** or to **View Update Details**. When a failed test requires review, new tests cannot be run on the site until the results have been approved or discarded through Autopilot.
+
+To resolve this issue remove the page from VRT settings or fix the redirect in the Dev environment for that page.
+  
+</Accordion>
+
+## Failed Deployment 
+<Accordion title="We could not deploy the updates to the Test or Live] environment due to an unexpected error." id="deploy-failed" icon="info-sign">
+
+### Issue
+
+Autopilot failed to deploy to Test or Live, however deploying to Dev from Multidev was successful. The most common reason for this is running clear cache or update db using drush or wp-cli failed after the code was deployed. Ensure that clearing the cache using Drush or the WP CLI works on the target environment.
+
+CSE can see in “Debug” tab why the workflow failed.
+   
+### Solution
+
+- If Drush/WP-CLI steps failed during diagnosis, resolve any errors thrown by the CMS
+
+- Run the deploy to test or live manually (the new code is already on master/dev env if we’ve made it this far). Autopilot will also attempt deploy again when the next round of updates is tested, but immediately running updates again will likely result in “UP TO DATE”, as the dev env already has the changes.
+
+If these steps fail, reach out to [Support](/support).
+  
+</Accordion>
+
+## Unreachable Site 
+
+<Accordion title="We ran into an unexpected issue with Autopilot because the site could not be loaded in the Autopilot Multidev or Dev environment." id="unreachable-site" icon="info-sign">
```

Comment:
```suggestion
<Accordion title="We ran into an unexpected issue with Autopilot because the site could not be loaded in the Dev environment." id="unreachable-site" icon="info-sign">
```

We should have a different error for the autopilot env, IMO.